public/index.html

```diff
@@ -130,6 +130,25 @@
       media="(device-width: 1024px) and (device-height: 1366px) and (-webkit-device-pixel-ratio: 2)"
       rel="apple-touch-startup-image"
     />
+    <style>
+      .LoadingMessage {
+        position: fixed;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        z-index: 999;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        pointer-events: none;
+      }
+      .LoadingMessage span {
+        background-color: rgba(255, 255, 255, 0.8);
+        padding: 0.8em 1.2em;
+        font-size: 1.3em;
+      }
+    </style>
```

Comment:
I put CSS and initial HTML message in `index.html` so that it's displayed & styled from initial HTML render, otherwise we'd need to wait on JS fetch and react init.